src/components/ProductsListContainer.js

```diff
@@ -0,0 +1,40 @@
+import productsData from '../mocks/en-us/featured-products.json';
+import Product from '../components/Product.js';
+import Paginator from './Paginator.js';
+
+const ProductsListContainer = (props) => {
+    const paginator = (props.hasPaginator !== undefined) ? <Paginator /> : <span/>;
+
+    const productsDataItems = productsData.results.map((item) => {
+        let itemToAdd;
+
+        if(props.productCategoryFilter === undefined) {
```

Comment:
As i mentioned before is you do something like 
```
if(!props.productCategoryFilter)
```
works exactly like this and it also works if the element is null